fuzzers/others/libafl-fuzz/src/afl_stats.rs

```diff
@@ -265,17 +298,33 @@ where
             .unwrap();
         let unstable_entries_in_map = unstable_entries_metadata.unstable_entries().len();
 
+        let auto_dict_entries = match self.autotokens_enabled {
+            false => 0,
+            true => state
+                .metadata::<Tokens>()?
+                .len()
+                .saturating_sub(self.provided_tokens),
+        };
         let stats = AFLFuzzerStats {
             start_time: self.start_time,
             last_update: self.last_report_time.as_secs(),
             run_time: self.last_report_time.as_secs() - self.start_time,
             fuzzer_pid: self.pid,
             cycles_done: queue_cycles,
             cycles_wo_find: self.cycles_wo_finds,
-            fuzz_time: 0,        // TODO
-            calibration_time: 0, // TODO
-            sync_time: 0,        // TODO
-            trim_time: 0,        // TODO
+            fuzz_time: state
+                .metadata::<FuzzTime>()
```

Comment:
You should only have a single metadata entry for the fuzzer and look that up once, if you add one metadata for each entry there's too much overhead